svn/trunk/html/embed.php

```diff
@@ -12,6 +12,27 @@
 $var = $GLOBALS['HYVOR_TALK_PLUGIN_JS_CONFIG'];
 ?>
 
+<?php if ($var['loadMode'] === 'click'): ?>
+
+    <div style="text-align:center">
+        <button id="hyvor-talk-load-button" onclick="this.style.display='none'">
+            <?php echo __('Load Comments') ?>
+        </button>
+    </div>
+
+    <script>
+        document.getElementById("hyvor-talk-load-button").addEventListener("click", function() {
+            const hyvorTalkComments = document.querySelector("hyvor-talk-comments");
+            hyvorTalkComments.load();
+        });
+    </script>
+
+<?php endif; ?>
+
+<div>
+    <hyvor-talk-comment-count page_id="<?php echo get_the_ID(); ?>"></hyvor-talk-comment-count>
```

Comment:
This seems like a mistake, no?

Reply:
Yes, it is. It's not in my working folder, seems I have missed to sync the repo folder.